common/frame.go

```diff
@@ -656,14 +656,14 @@ func (f *Frame) waitForSelector(selector string, opts *FrameWaitForSelectorOptio
 
 	ec := f.executionContexts[mainWorld]
 	if ec == nil {
-		return nil, fmt.Errorf("wait for selector %q cannot find execution context: %q", selector, mainWorld)
+		return nil, fmt.Errorf("wait for selector %q execution context is missing: %q", selector, mainWorld)
```

Comment:
The message is not grammatically correct now...

I think it should be consistent with the one in `waitForFunction`, so:
```suggestion
		return nil, fmt.Errorf("execution context %q not found", mainWorld)
```

Or `"execution context %q is missing"`, but it's the same lookup, so the messages should match.

And maybe this should be a panic. If it happens it's probably a bug on our side.